girder/girder_large_image/web_client/vue/components/CompositeChannels.vue

```diff
@@ -0,0 +1,184 @@
+        <table id="composite-channel-table" class="table table-condensed">
+            <thead>
+                <tr>
+                    <th class="channel-col">Channel</th>
+                    <th class="enabled-col">Enabled?</th>
+                    <th class="color-col">Color</th>
+                    <th class="precision-col">Min</th>
+                    <th class="precision-col">Max</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr
+                    v-for="channel in channels.filter(c => compositeChannelInfo[c] !== undefined)"
+                    :key="channel"
+                >
+                    <td>{{ channel }}</td>
+                    <td>
+                        <input
+                            type="checkbox"
+                            :value="channel"
+                            v-model="enabledChannels"
+                            @change="updateActiveChannels"
+                        >
+                    </td>
+                    <td :id="channel+'_picker'">
+                        <span
+                            class="current-color"
+                            :style="{ 'background-color': compositeChannelInfo[channel].falseColor }"
+                            @click="() => toggleColorPicker(channel)"
+                        />
+                        <color-picker
+                            class="picker-offset"
+                            v-if="colorPickerShown === channel"
+                            :value="compositeChannelInfo[channel].falseColor"
+                            @input="(swatch) => {updateChannelColor(channel, swatch)}"
+                        />
+                    </td>
+                    <td>
+                        <input
+                            type="number"
+                            step="0.01"
+                            min="0"
+                            max="1"
+                            :value="compositeChannelInfo[channel].min"
+                            @change.prevent="(event) => updateChannelMin(event, channel)"
+                        >
+                    </td>
+                    <td>
+                        <input
+                            type="number"
+                            step="0.01"
+                            min="0"
+                            max="1"
+                            :value="compositeChannelInfo[channel].max"
+                            @change.prevent="(event) => updateChannelMax(event, channel)"
+                        >
+                    </td>
+                    <td></td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</template>
+
+<style scoped>
+.current-color {
+    display: inline-block;
+    width: 50px;
+    height: 20px;
+    background-color: #000;
+    cursor: pointer;
+}
+.picker-offset {
+    position: absolute;
+    z-index: 100;
+    margin-left: 50px;
+}
+.table-container {
+    max-height: 200px;
```

Comment:
I think I'd rather have a bigger max-height here.  five channels requires scrolling, which is not that uncommon (the most I've seen is 60 channels).

Reply:
Changed in [560ec8e](https://github.com/girder/large_image/pull/1086/commits/560ec8e65cfdca9193753d0fdf427867352c3eb7)